### ISSUE TITLE
[Mod Suggestion] Deepslate From Lava

### ISSUE THREAD

**Issue by jingshenSN2** [TeamAOF/All-of-Fabric-5#150]
https://www.curseforge.com/minecraft/mc-mods/deepslate-from-lava
lava and water generate deepslate and cobbled_deepslate when y < 9, which allows players to build deepslate farming machines.

**Comment by k3nw4y** [comment]
![image](https://user-images.githubusercontent.com/46229190/153347875-e88f6000-b1d9-4c74-84ab-c0775b7f158f.png)
![image](https://user-images.githubusercontent.com/46229190/153348447-49a2d121-2420-425f-92e6-785ef3def5d3.png)

**Comment by dandin87** [comment]
Spectrum also has cobbled deepslate farming. Liquid crystal and lava make Cobbled Deepslate...
![2022-02-10_09 16 43](https://user-images.githubusercontent.com/9329117/153461007-a84af717-08dd-452e-bab0-117ccf95d874.png)

**Comment by jingshenSN2** [comment]
> This isn't to say your mod couldn't be added. Just to point out that there are other methods of farming cobbled deepslate.

yeah, I know. Just thought it's not ok to promote the mod without checking if there's similar functionality already provided by others.

**Comment by k3nw4y** [comment]
i just pointed out it because i thought you are not aware of  renewable/craftable deepslate
